composeApp/src/commonMain/kotlin/eu/heha/cyclone/ui/ComicReaderViewModel.kt

```diff
@@ -113,6 +113,21 @@ class ComicReaderViewModel(
         )
     }
 
+    fun setChapter(chapter: Chapter) {
```

Comment:
this is basically the same logic as when the function used to go to the previous and next chapter, please extract a function